.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
   - if [[ $coverage = 1 ]]; then mkdir -p build/logs build/cov; fi
   - if [[ $coverage = 1 ]]; then wget https://phar.phpunit.de/phpcov.phar; fi
   - if [[ $coverage = 1 ]]; then wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar; fi
+  - if [[ $lint = 1 ]]; then wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v1.12.2/php-cs-fixer.phar; fi
```

Comment:
> Because, for instance, we will not catch the issue with the removal of extra dots with the current setup.

I'm not sure I get what you mean. That is because of `-phpdoc_annotation_without_dot`, which I'll remove once #798 is merged.

> Semver is not relevant here because we don't use the internal API for the tool (only the config file)

The config file will break when using PHP-CS-Fixer 2.x because the package namespace has changed. There might be other breaking changes too.

> Anyway installing it through Composer (globally) instead of downloading the phar allows to download the last version of the 1.x branch.

Will do.